daml/Marketplace/Clearing/Role.daml

```diff
@@ -111,3 +111,20 @@ template Request
       Reject : ()
         do
           return ()
+
+template RequestRoleTermination
+  with
+    operator : Party
+    provider : Party
+    roleCid  : ContractId Role
+  where
+    signatory provider
+
+    controller operator can
+      TerminateAccept: ()
```

Comment:
I had to give these awkward names to avoid a name conflict with the `Accept` and `Decline` in the `Offer` template. Why does Daml not scope these?!

Reply:
I feel your pain, this gets very annoying very quickly... I used to use <template_name>_<choice_name> as a pattern to get around it but this also isn't great.

Reply:
Fortunately @alexgraham-da 's suggestion works, so the terminate choices can live on the Role contract itself without needing a separate contract!